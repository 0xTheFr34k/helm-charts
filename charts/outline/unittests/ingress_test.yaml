suite: test ingress

templates:
  - ingress.yaml

tests:
  - it: should be empty when ingress is not enabled
    asserts:
      - hasDocuments:
          count: 0

  - it: should match snapshot of default values
    set:
      ingress.enabled: true
      ingress.className: nginx-test
    release:
      name: outline
      namespace: outline
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    asserts:
      - matchSnapshot: { }
