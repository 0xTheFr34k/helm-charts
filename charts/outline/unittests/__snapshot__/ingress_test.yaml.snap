should match snapshot of default values:
  1: |
    apiVersion: networking.k8s.io/v1
    kind: Ingress
    metadata:
      labels:
        app.kubernetes.io/instance: outline
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: outline
        app.kubernetes.io/version: 1.0.0
        helm.sh/chart: outline-1.0.0
      name: outline
    spec:
      ingressClassName: nginx-test
      rules:
        - host: outline.mydomain.com
          http:
            paths:
              - backend:
                  service:
                    name: outline
                    port:
                      number: 3000
                path: /
                pathType: ImplementationSpecific
